styles/styles.css

```diff
@@ -445,6 +447,59 @@ main .section.banner .default-content-wrapper {
   text-transform: uppercase;
 }
 
+.btn {
```

Comment:
Why are these needed?

Reply:
for this: 
<img width="519" alt="image" src="https://github.com/hlxsites/hsf-commonmoves/assets/53608019/721f795f-e8a6-42ae-8f4b-e049418d0861">
Felt these could be candidates for global primary and secondary buttons...as I didn't see such styling already implemented.

Reply:
There is a primary button global definition, it has a specific dom structure `p.button-container > a`.

There isn't a secondary one, but that can be achieved with a specific dom structure as well `p.button-container > em > a`
This will add `button secondary` classes to the anchor, we would just need to add the CSS to create the ux of the button on the right.

Reply:
Right, but even though this isn't authorable doesn't mean we can't still generate the same markup as the authored styles, so that we use the same, existing CSS